src/axisym_displ_based_foeppl_von_karman/axisym_displ_based_fvk_elements.cc

```diff
@@ -289,23 +276,40 @@ namespace oomph
           this->raw_nodal_value(l, nodal_index_fvk(2)) * dpsidr(l, 0);
       } // End of loop over nodes
 
-      // Compute the stresses:
-      //---------------------
-      sigma_r_r =
-        1.0 / (1.0 - nu_local * nu_local) *
-        (interpolated_du_rdr + 0.5 * interpolated_dwdr * interpolated_dwdr +
-         nu_local * 1.0 / interpolated_r * interpolated_u_r);
-
-      sigma_phi_phi =
-        1.0 / (1.0 - nu_local * nu_local) *
-        (1.0 / interpolated_r * interpolated_u_r +
-         nu_local *
-           (interpolated_du_rdr + 0.5 * interpolated_dwdr * interpolated_dwdr));
-
-      // Success!
-      return true;
+      // The centre stress must use a different analytical form to avoid
+      // dividing by zero.  It can be found from the form below by assuming:
+      // a) u_r=0 at the origin (required under axisymmetry unless there is a
+      // puncture) which leads to the term u_r/r -> du_r/dr as r -> 0 (by
+      // L'Hopital's rule), and
+      // b) given dw/dr=0 at the origin (required for axisymmetry as there can
+      // be no kink in a plate).
+      // Note that this process results in s_{rr}=s_{\phi\phi} at the origin
+      // which is expected as the two coordinate directions are
+      // indistinguishable at the origin under axisymmetry.
+      // [zdec] Should this be interpolated_r < tolerance to prevent floating
```

Comment:
I think it's fine; the error really only occurs when r is exactly zero. Just change comment to:
"Yes, this really does require a floating point comparison since the division by zero only occurs when the radius is exactly zero."